MapsAndPlacesDemo/MapsAndPlacesDemo/apikeys.swift

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/// API keys needed for this project
+public enum apikeys {
```

Comment:
Let's make this camelcase (i.e. `ApiKeys`). File name should also be camelcased.